v0/pipeline/steps/cd/jenkins-build-step-info.yaml

```diff
@@ -40,6 +43,9 @@ required:
 properties:
   connectorRef:
     type: string
+  consoleLogPollFrequency:
+    type: string
```

Comment:
DOn't think you need it here

Reply:
Similar fields were added here also so i added

Reply:
Why we are duplicating fields here?

Reply:
I think they copied from the dynamic jsons that were present and there these were duplicated.
For now it is fine but if we create a new step and stage and if it is not needed we might want to avoid them